backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/HybridBackupCommand.java

```diff
@@ -314,4 +324,12 @@ private Guid createVmBackup(Guid snapshotId) {
         persistCommandIfNeeded();
         return vmBackup.getId();
     }
+
+    private boolean isAnyDiskLocked() {
+        if (getParameters().getVmBackup().getDisks() != null) {
+            return getParameters().getVmBackup().getDisks().stream()
+                    .anyMatch(disk -> disk.getImageStatus() == ImageStatus.LOCKED);
+        }
+        return false;
```

Comment:
we should never get to the `return false;` part, right? if so, let's add a log message to indicate that we face an unexpected state or rely on having the disks != null